.docs/mixpanel-chart-explanations.md

```
@@ -0,0 +1,377 @@
+**What it shows:** Pie chart of the HTTP response codes the API returned.
+
+```mermaid
+pie showData
+    title Healthy API status mix
+    "2xx success" : 96
+    "4xx client error" : 3
+    "5xx server error" : 1
+```
+
+**Categories:**
+| Code | Meaning |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)